exercises/exercise3.js

```diff
@@ -1,12 +1,56 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Would you rather!',
+    prompt: 'Would you rather go shopping or out to eat?',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'its 2020, solid food is for the weak! lets go shopping';
     },
+    isCaseSensitive: false,
+    isReply: true,
+    isLitening: true
   },
+
+  {
+    label: 'Would you rather the aliens that make first contact be organics or robots',
+     prompt: 'I would rather #{name} make first contact. ',
+      handler: function (params) {
+        
+        if ( params.name === "robots"  ){
+          return " really?" + params.name1 + "would be my last choice"
+        }
+        else  {
+          return "See your smart!";
+        }
+      },
+      isCaseSensitive: false,
+      isReply: true,
+      isLitening: true
+    },
+      {
+        label: "Dance break",
+        prompt:"#{playerTwo} challanges #{playerOne} to a dance off, who will win?",
+        handler: function (params) {
+          if (params.playerTwo > playerOne) {
+            return params.playerOne + "cant move like me!";
+          } 
+          else if (params.playerOne === params.playerTwo) {
+            return "Oh dang, I got a cramp, lets go again!";
+
+          } 
+          else {
+            return params.playerTwo + " take your beating with some pride!";
+
+          }
+          
+        }
+
+      },
+      
+      {
+
+      }
+
 ];
```

Comment:
These are great! Any fixes needed were minor and your syntax overall is looking good! With tabs and spaces, there are some needed or too many here and there, but I still really like what I'm seeing.
/// Tech: 0 - Does not meet expectations. The ONLY reason for this is because we asked for 7 - 10 interactions on this one. If not for that, I would give you a meet expectations as the code in the four interactions you did submit is great!! (p.s. I'm counting the 'Tell Story' prompt you have in your exercise2.js file in this)
/// Creativity: 2 - Exceeds expectations, you wonderful creature, you! I LOVE the prompts you did. Especially the responses to the challengers duking it out over name length! But that robot response tho - haha. Well done and well thought out here.
/// Code Quality: 1 - Meets expectations, good job! Again, like I said, any fixes were minor and you show an overall grasp of the concepts we've learned so far. Really, you did a great job here! I'd work on catching those spaces and tabs, but other than that visually your code looks good.
/// Total: 3 - Pass. You did it! Great job!